### PR TITLE
Adds a count to the file name in the download manager if the user dow…

### PR DESCRIPTION
**Pull request by VikasShashidhar**
Adds a count to the file name in the download manager if the user downloads a file with the same name.

This mimics the Google Chrome behaviour.

@lneir @VishwasShashidhar Please review.

### CODE REVIEW

File: js/downloadManager/index.js [hunk 1]
```diff
@@ -194,4 +195,33 @@ function initiate() {
             });
         }
     }
+}
+
+/**
+ * Return a file display name for the download item
+ */
+function getFileDisplayName(fileName) {
```
Comment (lneir) [inline js/downloadManager/index.js]: what about case where i download file.  
then refresh client.
now download again.
 
the download filename should reflect the actual version in the downloads folder - that is what chrome is doing.

Comment (VikasShashidhar) [reply js/downloadManager/index.js]: Chrome persists all the data (file name, URL, etc.) that we download and makes it available to us. Currently, both the Paragon and the Electron Wrapper don't have such an option.

Comment (VikasShashidhar) [reply js/downloadManager/index.js]: Waiting for input from Product -> https://perzoinc.atlassian.net/browse/ELECTRON-146

Comment (VikasShashidhar) [reply js/downloadManager/index.js]: @lneir After discussions with Thibaud last week, can we consider this as a minor issue and merge this PR? We already have an open ticket to track the persistent behaviour of download bar.

**Pull request merged**